.github/workflows/ci.yml

```diff
@@ -41,6 +41,18 @@ jobs:
         run: |
           git config --global core.bigfilethreshold 100m
 
+      - name: Install Forc
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: forc
```

Comment:
You should fix a `forc` version here instead of using latest

Reply:
Wait isn't it going to be problematic? I feel like the whole point was to use the last compiler version (ie last `forc` version) so we shouldn't pin it?

Reply:
Yeah, ideally we want to test it against the latest. But, for instance, if for some reason we don't wanna use the latest (e.g. latest breaks something on the SDK, we're aware of it, but we're still working on a fix), we might not want to block other PR's CI pipeline because of it. So having this control is nice. 

But, then again, we have to watch ourselves and make sure we're updating this version when there's a new `forc` version.

Reply:
If we want the best of both worlds, then we can create a second CI workflow that runs against `latest` `forc` and isn't required to pass to merge PRs. Then if it breaks we know, but it's not blocking anything.

Reply:
Ok, I feel like we should find a way to ensure this?

Reply:
Not a huge fan of that because I believe it would increase the CI runtime considerably :(. We're currently sitting at ~10min, running another one, in this case, would mean building all Sway projects again.

Reply:
maybe the solution would be to run the `fuels-rs` CI on every `forc` major release?
But just on releases?

Reply:
Ah no wait that doesn't work